pages/_app.js

```diff
@@ -90,6 +92,18 @@ export default function App({ Component, pageProps }) {
     setFilteredMovies(sorted);
   };
 
+  const handleAddToFavorites = (movieId) => {
+    const movieToAdd = Movies.find((movie) => movie.id === movieId);
+    if (!favorites.some((movie) => movie.id === movieToAdd.id)) {
+      setFavorites([...favorites, movieToAdd]);
+    } else {
+      const updatedFavorites = favorites.filter(
+        (movie) => movie.id !== movieId
+      );
+      setFavorites(updatedFavorites);
+    }
+  };
+
   return (
     <>
```

Comment:
Dont you want to use different pages? For me it would be more logical to put the Favorite list on a different page instead of using all at one page together.

Also i saw in the deployment, that the favorit movies arent added to the favorite list.

Reply:
right now im just trying to get it working I might create it on different page later